components/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+interface AccordionItem {
+  question: string;
+  answer: string;
+}
+
+interface AccordionProps {
+  items: AccordionItem[];
+}
+
+const Accordion: React.FC<AccordionProps> = ({ items }) => {
+  const [activeIndex, setActiveIndex] = useState<number | null>(null);
+
+  const handleToggle = (index: number) => {
+    setActiveIndex((prevIndex) => (prevIndex === index ? null : index));
+  };
+
+  return (
+    <div>
+      {items.map((item, index) => (
+        <div
+          key={index}
+          className={`overflow-hidden transition-max-height border-t-2 ${
+            activeIndex === index ? 'max-h-96' : 'max-h-20'
+          }  ${index === items.length - 1 ? 'border-b-2' : ''}`}
+        >
+          <div
+            className='flex justify-between items-center p-4 cursor-pointer'
```

Comment:
Can we use the icon '+' (plus) instead of the triangle?

Reply:
Updated this